src/main/kotlin/com/cognifide/gradle/aem/pkg/ComposeTask.kt

```diff
@@ -176,8 +180,12 @@ open class ComposeTask : Zip(), AemTask {
     fun includeBundles(project: Project) {
         dependProject(project)
 
+        val extraProperties = project.extensions.extraProperties
+        val bundlePath: String = if (extraProperties == null || !extraProperties.has("bundlePath")) config.bundlePath
+        else extraProperties.get("bundlePath") as String
+
         bundleCollectors += {
-            JarCollector(project).all.toList()
+            Pair(bundlePath, JarCollector(project).all.toList())
```

Comment:
Ok, maybe tomorrow if I will find some time, I will illustrate that to you; but the case with includeBundle with different path than configured in sub project is not yet supported but I am planning to implement it in a similar way as you did. 

What is more, Gradle is such elastic so that you are not forced to extend plugin to achieve what you want, because aemCompose / includeProject is using from() method from ZIP task which ComposeTask is extending, so that ... you can use that `from(xx {spec.into('bundle-Path'}) construction to cover your specific case